src/profile-logic/gecko-profile-versioning.js

```diff
@@ -430,5 +430,30 @@ const _upgraders = {
     }
     convertToVersionElevenRecursive(profile);
   },
+  [12]: profile => {
+    //This version will add column numbers to the JS functions and scripts.
```

Comment:
Tiny style nit: Generally we have a space after the `//` in comments in this project.